### PR TITLE
chore: migrate docs job to Trampoline V2

### LINKED ISSUE #64: migrate `docs` Kokoro job to Trampoline V2
This allows us to run the build locally. I need this feature for further developing new doc pipeline.

### PR DESCRIPTION
**Pull request by tmatsuo**
This allows us to run the whole job locally.

Example:

```bash
$ TRAMPOLINE_IMAGE=gcr.io/tmatsuo-test/python-lib-docs \
  TRAMPOLINE_DOCKERFILE=.kokoro/docker/docs/Dockerfile \
  TRAMPOLINE_BUILD_FILE=.kokoro/publish-docs.sh \
  STAGING_BUCKET=tmatsuo-test-bucket \
  .kokoro/trampoline_v2.sh
```

You need to copy the service account with the permission on the bucket
to `/dev/shm/73713_docuploader_service_account` for the above command to
work.

fixes #64 

CC @tbpg

### CODE REVIEW

**Comment by tmatsuo** [comment]
I run the `docs` Kokoro job against my branch with `STAGING_BUCKET=tmatsuo-test-bucket` and it worked.
[Build log](https://source.cloud.google.com/results/invocations/8caaa4e3-e1c9-48c7-84ae-b353609cc86d/log)

File: noxfile.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def cover(session):
     session.run("coverage", "erase")
 
 
-@nox.session(python=DEFAULT_PYTHON_VERSION)
+@nox.session(python="3.8")
```
Comment (busunkim96) [inline noxfile.py:138]: Does this need to be 3.8 for this to work?

Comment (tmatsuo) [reply noxfile.py:138]: I simplified the Docker image which now only has Python 3.8. We can revert back to the old image and use Python 3.7, or install Python 3.7 then we can use the Default Python.

Comment (tmatsuo) [reply noxfile.py:138]: I'll modify the Dockerfile so that we can use the DEFAULT_PYTHON_VERSION.

Comment (tmatsuo) [reply noxfile.py]: @busunkim96 Done

**Comment by tmatsuo** [comment]
I think it's cleaner to add the environment variable in the Kokoro configs. I'll revisit it.

**Comment by tmatsuo** [comment]
I run the docs Kokoro job against my branch with STAGING_BUCKET=tmatsuo-test-bucket again and it worked.

[Build log](https://source.cloud.google.com/results/invocations/b626c456-8c28-4640-8af0-460a06757682/targets)

**Comment by tmatsuo** [comment]
I manually checked the HTML content created by the following command:

```sh
$ TRAMPOLINE_IMAGE=gcr.io/tmatsuo-test/python-lib-docs \
  TRAMPOLINE_DOCKERFILE=.kokoro/docker/docs/Dockerfile \
  TRAMPOLINE_BUILD_FILE=.kokoro/publish-docs.sh \
  STAGING_BUCKET=tmatsuo-test-bucket \
  .kokoro/trampoline_v2.sh
```

and they look ok, merging.

**Pull request merged**